frontend/src/app/order-table/order-table.component.ts

```diff
@@ -28,4 +28,14 @@ export class OrderTableComponent implements OnInit {
     this.dialog.open(TrackingNumberComponent, dialogConfig);
   }
 
+  releaseOrder(order : number): void{ 
+    this.orderControllerService.releaseOrder(order).subscribe();
+    window.location.reload();
+  }
+
+  claimOrder(order: number):void{
+    this.orderControllerService.assignToMe(order).subscribe();
+    window.location.reload();
```

Comment:
Same as the above, but with other status, and update the 'assignedTo' field to the currently signed in user